features/facebookImport/test/utils/analyses-execution.js

```diff
@@ -7,7 +7,7 @@ export async function runAnalysisForExport(
     zipFile,
     pod = new MockerPod()
 ) {
-    const facebookAccount = await importZip(zipFile);
+    const facebookAccount = await importZip(zipFile, pod);
```

Comment:
Not totally comfortable with the fact that something called "importZip" returns a `facebookAccount`. But I guess it's a nitpick.